src/routes/+page.svelte

```diff
@@ -1,17 +1,21 @@
 <script>
 	// export let data;
-	import { Header, HeroSlider, Footer, Button } from '$lib/index.js';
+	import { Header, HeroSlider, Footer, AgendaCarousel } from '$lib/index.js';
 </script>
 
 <Header />
 <HeroSlider />
 <Footer />
-<Button
-	buttonText="Bekijk de Bieb"
-	buttonBackground="#4E7141"
-	svgFill="#F9E9C2"
-	buttonColor="#F9E9C2"
+<AgendaCarousel
+	agendaItems={[
```

Comment:
Please put the object literal inside the ```<script>``` section and use a ```const``` name to refer to it..

Reply:
changed